Gems/Multiplayer/Code/Include/Multiplayer/Components/SimplePlayerSpawnerComponentBus.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Outcome/Outcome.h>
+#include <AzCore/std/string/string.h>
+
+namespace Multiplayer
+{
+    //! @class SimplePlayerSpawnerRequests
+    //! @brief The SimplePlayerSpawnerRequest event-bus exposes helper methods regarding network player spawners.
+    //! Although the Multiplayer System automatically spawns in players, it's common for game specific server logic to retrieve valid spawn locations when respawning a player.
+    class SimplePlayerSpawnerRequests : public AZ::EBusTraits
+    {
+    public:
+        static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Single;
```

Comment:
Single/Single kind of cries out for an Interface.. This is also a little extra weird since it's already inheriting from the IMultiplayerSpawner interface, and all your use-cases appear to be native code.

Reply:
I was thinking that we didn't want to lump in any assumptions about iMultiplayerSpawner besides the responsibility of spawning a player OnPlayerJoin. The original custom code for MPS used a separate IMultiplayerSpawner and IPlayerSpawner (for having spawn points and round-robin indexing); this carries the tradition of separation.

Reply:
Fixed! Is now an AZ::Interface.